jenkins-builds/production.sh

```diff
@@ -10,18 +10,20 @@ scriptdir=$(dirname $0)
 
 usage() {
     echo -e "\n Usage: $0 [OPTIONS] -l <list> -p <prefix>
-    -a,--arch     Architecture (gpu or mc)           (mandatory: Dom and Piz Daint only)
-    -f,--force    Force build of item(s) in list     (optional: double quotes for multiple items)
-    -h,--help     Help message
-    -l,--list     Absolute path to production file   (mandatory: EasyBuild production list)
-    -p,--prefix   Absolute path to EasyBuild prefix  (mandatory: installation folder)
-    -u,--unuse    Module unuse colon separated PATH  (optional: default is null)
-    -x,--xalt     [yes|no] update XALT database      (optional: default is yes)
+    -a,--arch         Architecture (gpu or mc)           (mandatory: Dom and Piz Daint only)
+    -f,--force        Force build of item(s) in list     (optional: double quotes for multiple items)
+    -h,--help         Help message
+    -l,--list         Absolute path to production file   (mandatory: EasyBuild production list)
+    -p,--prefix       Absolute path to EasyBuild prefix  (mandatory: installation folder)
+    -u,--unuse        Module unuse colon separated PATH  (optional: default is null)
+    -x,--xalt         [yes|no] update XALT database      (optional: default is yes)
+    --hide-deps       Hide the hidden dependencies
```

Comment:
if we call `eb file.eb`, then `file.eb` is not actually a dependency. How about:

- Force hide modules listed in 'hide-deps' (TestingEB only)